clifford/numba/_multivector.py

```diff
@@ -306,3 +306,21 @@ def impl(self, arg):
             mv.value[inds] = self.value[inds]
             return mv
         return impl
+
+
+@numba.extending.overload_method(MultiVectorType, 'mag2')
+def MultiVector_mag2(self):
+    def impl(self):
+        return (~self * self).value[0]
```

Comment:
This one we can't reuse because of the optimizations made in the pure-python implementation...